ood-portal-generator/lib/ood_portal_generator/view.rb

```diff
@@ -23,6 +23,11 @@ def initialize(opts = {})
       @map_fail_uri     = opts.fetch(:map_fail_uri, nil)
       @pun_stage_cmd    = opts.fetch(:pun_stage_cmd, "sudo /opt/ood/nginx_stage/sbin/nginx_stage")
 
+      # Maintenance configuration
+      @use_maintenance          = opts.fetch(:use_maintenance, false)
+      @maintenance_ip_whitelist = opts.fetch(:maintenance_ip_whitelist, [])
```

Comment:
perhaps its preferable to coerce to Array here? i.e. `Array(opts.fetch(:maintenance_ip_whitelist, []))`

Reply:
Change made